Kerberos.NET/Crypto/Pkinit/PKInitString2Key.cs

```diff
@@ -1,69 +1,111 @@
 ﻿using System;
+using System.Diagnostics;
+using System.Security.Cryptography;
 
 namespace Kerberos.NET.Crypto
 {
     public static class PKInitString2Key
```

Comment:
I don't know whether this class is "hot" or not, but there where some allocations that can be avoided easily. That's why I touched this class.